src/vc_util/src/lib.rs

```diff
@@ -327,16 +327,14 @@ fn validate_claim<T: PartialEq<S> + std::fmt::Display, S: std::fmt::Display>(
         if expected == actual {
             Ok(())
         } else {
-            ic_cdk::println!(
+            println!(
```

Comment:
Do you want to keep this in? Usually, it's not expected for a library to print to stdout...

Reply:
I'll remove these once I can add custom error messages to returned errors (which currently require `static` strings).
Added https://dfinity.atlassian.net/browse/GIX-2190 to address this issue.